OONIProbeUITests/OONIProbeUITests.swift

```diff
@@ -45,7 +45,7 @@ class OONIProbeUITests: XCTestCase {
         snapshot("02TestResults")
         
         let thepiratebayStaticText = tablesQuery.staticTexts["http://thepiratebay.org"]
-        tablesQuery/*@START_MENU_TOKEN@*/.cells.containing(.image, identifier:"websites")/*[[".cells.containing(.staticText, identifier:\"1 blocked\")",".cells.containing(.staticText, identifier:\"Websites\")",".cells.containing(.staticText, identifier:\"12\/12\/18, 1:23 AM\")",".cells.containing(.image, identifier:\"websites\")",".cells.containing(.staticText, identifier:\"39 tested\")"],[[[-1,4],[-1,3],[-1,2],[-1,1],[-1,0]]],[1]]@END_MENU_TOKEN@*/.staticTexts["AS30722 - Vodafone Italia S.p.A."].tap()
+        tablesQuery.cells.containing(.staticText, identifier:"Websites").staticTexts["AS30722 - Vodafone Italia S.p.A."].tap()
```

Comment:
What's `/*@START_MENU_TOKEN@*/`? It seems the code here is doing something different?

Reply:
Auto generated XCode code. I was seeing only .image etc